.github/workflows/lint.yml

```
@@ -26,4 +26,23 @@ jobs:
+        with:
+          node-version: '22'
+          cache: 'npm'
+      - name: Install dependencies
+        working-directory: apps/web
+        run: npm install
+      - name: Run ESLint
+        working-directory: apps/web
+        run: npm run lint
+      - name: Check for ESLint errors
+        if: ${{ failure() }}
+        run: echo "ESLint failed!" && exit 1
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)